modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -7045,9 +7045,11 @@ feature.flag.LPS-135430.title=Proxy Object
 feature.flag.LPS-179669.description=Search the company index for matching content. Add facets with translated display terms and set a search blueprint in the request to add custom searching, filtering, aggregation, and sorting.
 feature.flag.LPS-179669.title=Search Headless API
 feature.flag.type.description.beta=Beta features are experimental and not supported. Use with caution.
+feature.flag.type.description.deprecation=Deprecated features will be removed from DXP soon. They might be replaced with a new and better alternative.
```

Comment:
@ethib137, Mateo provided in private chat, I guess I should've added to the story.

Reply:
Yep, adding it to the story is helpful. It's also a good idea to try and not have those discussions in a private chat. It's much better to have that discussion as comments on the ticket so that the history is publicly available for others to understand. 😀

Reply:
Great, I'll keep public discussions from now one.

Reply:
(for history purposes)

Added the description to the [story](https://liferay.atlassian.net/browse/LPS-189291) with a slightly change. 

"Deprecated features will be removed soon. They might be replaced with a new and better alternative."